package.json

```diff
@@ -20,6 +21,11 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
-  }
+    "eject": "react-scripts eject",
+    "dev": "nodemon ."
```

Comment:
Doesn't need to be here for react app, already have start.

Reply:
Will also fix that.
Thnx Good find.